README.md

```diff
@@ -0,0 +1,141 @@
+<div align="center">  <br/>
+  <h1><b> Morse code decode
+</b>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Morse code decode <a name="about-project"></a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Stack</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Decode Morse code code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby installed and configured properly. [Guide](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/BHObserver/Morse-Code-Decoder.git
+```
+
+### Usage
+
+In the project folder run the following
+
+```
+bundle install
+
+gem install rubocop
+```
+
+To run the program you have to run
+
+```
+ruby app.rb
+```
+
+### Authors
+
+👤 **Burhan Uddin**
+
+- GitHub: [Burhan Uddin](https://github.com/BHObserver/)
+- Twitter: [Burhan Uddin](https://twitter.com/BurhanU14173360)
+- LinkedIn: [Burhan Uddin](https://www.linkedin.com/in/bhobserver/)
+
+👤 **Kenneth Adutwum**
+
+- GitHub: [@githubhandle](https://github.com/Kwamay)
+- Twitter: [Kwamay3](https://twitter.com/Kwamay3)
+- LinkedIn: [Kwamay Adu](https://linkedin.com/in/кωαмαу-adu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add tests**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/BHObserver/Morse-Code-Decoder/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to reach out to us.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
It would be nice to add correct information about the school library project in your readme file what Im seeing is information about decording morse code please lets be professional.